articles/client-platforms/react/05-linking-accounts.md

```diff
@@ -0,0 +1,382 @@
+---
+title: Linking Accounts
+description: This tutorial will show you how to integrate Auth0 with ReactJS to link accounts.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/04-Linking-Accounts',
```

Comment:
@flaviogranero 
Please update the sample URL to: https://github.com/auth0-samples/auth0-react-sample/tree/master/05-Linking-Accounts